Add prow presubmit configuration for building components (#81)
kyma-project/test-infra: development/README.md

## Installation

1. Set an OAuth2 token that has the read and write access to the bot account. You can set it either as an environment variable named `OAUTH` or interactively during the installation.
   To generate a new token, go to the **Settings** tab of a given GitHub account and click **Developer Settings**. Choose **Personal Access Token** and **Generate New Token**.
   In the new window, select all scopes and click **Generate token**.

   > **NOTE:** Create a separate bot account instead of using your personal one. If the Prow bot account is the same as the account that creates a job-triggering comment, the job is not triggered.

2. Export the environment variables:

- **BUCKET_NAME** is a GCS bucket in the Google Cloud project that is used to store Prow Secrets.
- **KEYRING_NAME** is the KMS key ring.
- **ENCRYPTION_KEY_NAME** is the key name in the key ring that is used for data encryption.

> **NOTE:** For more details on setting up Google Cloud Project refer to [Prow Secrets management](https://github.com/kyma-project/test-infra/blob/master/docs/prow-secrets-management.md)

The Prow installation script assumes that the following service accounts exist:

> - **sa-gke-kyma-integration** with a role allowing to create Kubernetes clusters

Reviewer: ```suggestion
- **sa-gke-kyma-integration** with a role that allows the account to create Kubernetes clusters.
```

> - **sa-vm-kyma-integration** with roles allowing to provision Virtual Machines

Reviewer: ```suggestion
- **sa-vm-kyma-integration** with roles that allow the account to provision virtual machines.
```

> - **sa-gcs-plank** with roles allowing to store in Bucket

Reviewer: ```suggestion
- **sa-gcs-plank** with roles that allow the account to store objects in a Bucket.
```

> - **sa-gcr-push** with roles allowing to push to Google Container Repository

Reviewer: ```suggestion
- **sa-gcr-push** with roles that allow the account to push images to Google Container Repository.
```

> with their keys, encrypted using the **ENCRYPTION_KEY_NAME** from **KEYRING_NAME**, being present in \*\*BUCKET_NAME\*\*

Reviewer: ```suggestion
The account files are encrypted with the **ENCRYPTION_KEY_NAME** key from **KEYRING_NAME** and are stored in **BUCKET_NAME**.
```
